composer.json

```diff
@@ -41,7 +41,6 @@
         "symfony/validator": "^3.4",
         "symfony/yaml": "^3.4",
         "twig/extensions": "^1.5",
-        "twig/twig": "1.*",
```

Comment:
Can we make the dependency on `twig/twig` explicit?

Reply:
Sure, but we did more or less the same with Swiftmailer (depend on the bundle instead of bundle + library).

Do you want me to change it?

Reply:
I think it would be nice to keep it as explicit dependency.